src/components/drawer/Drawer.tsx

```diff
@@ -534,6 +534,7 @@ const Drawer = ({
       >
         <SafeAreaView
           edges={['bottom', ...((isFullscreen ? ['top'] : []) as Edge[])]}
+          style={{ height: isFullscreen ? '100%' : 'auto' }}
```

Comment:
Nit: I think it might be better practice to have two different classes declared within createStyles and pick one based on the isFullScreen boolean so as to keep all styling in one place and not spread out throughout the file

Reply:
I like it! Updated